### ISSUE TITLE
post to getlektor.com

### ISSUE THREAD

**Issue by nixjdm** [rlaverde/lektor-static-search#4]
Hey @rlaverde! I found your plugin on pypi. I didn't know you had made one! Can you make a pull request to getlektor.com adding it to the plugin list? You can read about how to do that here: https://www.getlektor.com/docs/plugins/publishing/

I hope you do!

Thanks!

**Comment by rlaverde** [comment]
Yes, I was planning to do that, but I hadn't do it because I wanted to add a template tag to easy integrate Lurn.js

Anyway I'll add it to lektor puglin list, and try to add the missing feature soon

**Comment by webknjaz** [comment]
FWIW I think it'd be useful to be listed even w/o said feature...
